sdk/swagger.json

```diff
@@ -1187,8 +1196,22 @@
         "workspaceCollectionName": {
           "type": "string",
           "description": "The workspace collection name"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The workspace display name"
         }
       }
+    },
+    "CreateWorkspaceRequest": {
+      "type": "object",
+      "description": "A Create Workspace Request",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The workspace name"

```

Comment:
This is just a friendly name, right?
Do we support OData filter for this property when getting all available workspaces in wc?